lms/djangoapps/courseware/masquerade.py

```diff
@@ -103,7 +127,7 @@ def get(self, request, course_key_string):
                 'course_key': course_key_string,
                 'group_id': course.group_id,
                 'role': course.role,
-                'user_name': course.user_name or ' ',
+                'user_name': course.user_name or None,
```

Comment:
Confirmed from previous comment, we no longer need to be tied to this old convention;
we're the only ones using this endpoint.

This makes state management easier (ie, we don't have to say `userName.trim()`).